apps/metric_vnode/eqc/metric_vnode_eqc.erl

```diff
@@ -206,21 +203,21 @@ prop_handoff() ->
                    List6 = [true || {_V, _V} <- List2],
 
                    {async,{fold, AsyncHc, _}, _, C2} =
-                       ?V:handle_handoff_command(FR, self(), C1),
+                       metric_vnode:handle_handoff_command(FR, self(), C1),
                    Lc = AsyncHc(),
                    Lc1 = lists:sort(Lc),
                    {async,{fold, Async, _}, _, _} =
-                       ?V:handle_coverage({metrics, ?B}, all, self(), S1),
+                       metric_vnode:handle_coverage({metrics, ?B}, all, self(), S1),
```

Comment:
Line 210 is too long:                        metric_vnode:handle_coverage({metrics, ?B}, all, self(), S1),.